### PR TITLE
Add more actions when proposal owned by you

### PR DESCRIPTION
**Pull request by michalsidzej**
Screenshot 2022-05-05 at 09 17 49

### CODE REVIEW

File: packages/frontend/src/pages/positions/PositionDetails.tsx [hunk 1]
```diff
@@ -99,6 +133,7 @@ export function PositionDetails({
   transactions,
   account,
 }: PositionDetailsProps) {
+  const ownedByYou = true
```
Comment (sz-piotr) [inline packages/frontend/src/pages/positions/PositionDetails.tsx:136]: ethAddress === account

File: packages/frontend/src/pages/positions/PositionDetails.tsx [hunk 2]
```diff
@@ -1,35 +1,69 @@
...
-  price,
-}: PositionDetailsProps['assets'][number]) => ({
-  cells: [
-    <AssetNameCell assetId={assetId} />,
-    balance.toString(),
-    price ? `${formatUSDCents(price)}` : '-',
-    formatUSDCents(totalUSDCents),
-  ],
-})
+}: Pick<PositionDetailsProps['assets'][number], 'assetId' | 'balance'>) => {
```
Comment (sz-piotr) [inline packages/frontend/src/pages/positions/PositionDetails.tsx:37]: why not just `string`

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/frontend/src/pages/positions/PositionDetails.tsx (hunk 2 above)
Comment (michalsidzej) [reply packages/frontend/src/pages/positions/PositionDetails.tsx]: This is not the return value, but arguments. Simplyfied it a little bit for sake of readability

File: packages/frontend/src/pages/positions/PositionDetails.tsx [hunk 3]
```diff
@@ -1,35 +1,72 @@
...
+const balanceTableColumns = (ownedByYou: boolean) => {
+  const columns: Column[] = [
+    { header: 'Name' },
+    { header: 'Balance', numeric: true },
+    { header: 'Unit price', numeric: true },
+    { header: 'Value', numeric: true },
+  ]
+
+  if (ownedByYou) {
+    columns.push({ header: 'Forced', maxWidthClass: 'w-0' })
```
Comment (sz-piotr) [inline packages/frontend/src/pages/positions/PositionDetails.tsx:24]: Rename `maxWidthClass` to `className`

File: packages/frontend/src/pages/positions/PositionDetails.tsx [hunk 4]
```diff
@@ -1,35 +1,72 @@
...
-    balance.toString(),
-    price ? `${formatUSDCents(price)}` : '-',
-    formatUSDCents(totalUSDCents),
-  ],
-})
+}: {
+  assetId: AssetId
+  balance: bigint
+}) => {
+  if (AssetId.symbol(assetId) === 'USDC' && balance !== 0n) {
```
Comment (sz-piotr) [inline packages/frontend/src/pages/positions/PositionDetails.tsx:41]: Add a constant to AssetId `AssetId.USDC`

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**